docs/quickstarts/twitter.rst

```diff
@@ -54,9 +54,49 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
+Visit http://127.0.0.1:5000 in your browser, and you should start the OAuth dance
 immediately.
 
+In your view, make sure you use a plain hyperlink to begin the dance so that the redirect loads Twitter's confirmation page in the browser. 
+
+```
+<a href="/twitter/login">Sign in with Twitter</a>
+```
+
+You can only use http libraries like axios to check the authentication status. 
+
+For instance, when the component mounts, you can call a function to check the authentication status. 
+
+If the authentication fails, then you can display a hyperlink for the user to begin the dance. 
+
+If it succeeds, then simply display the authenticated username. 
+
+```
+// an example using axios and vuejs but this could be ported to react, angular, or vanilajs
+// create 3 state variables: welcome (String ""), authenticated (Boolean false), authenticateCheckComplete (Boolean false)
+// if (!authenticated && authenticateCheckComplete) show the hyperlink 
+// if (authenticated && authenticateCheckComplete) show the username 
+
+const url = (document.domain === "127.0.0.1") ? 'http://127.0.0.1:5000/twitter/auth' : 'https://your-production-domain/twitter/auth'
+
+function checkAuthentication(){
+    const self = this;
+
+    axios.get(url).then( 
+        response => {
+            if (response.data.screen_name) {
+                self.welcome = "welcome " + response.data.screen_name;
+                self.authenticated = true;
+            }
+        }
+    ).catch(error => {
+        this.errored = error
+    }).finally(() => self.authenticateCheckComplete = true);
+
+}
+
+```
+
```

Comment:
This document is not Markdown, but ReStructuredText. As such how to do code blocks etc is quite different and this now renders rather broken documentation. Feel free to click on the "View File" button to see what I mean.

Please take a quick look at https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html and the rest of our documentation on how to do this correctly.

Reply:
@daenney Ok when I click View File now - the file seems to be rendering correctly